### DISCUSSION TITLE
Future3 usb buttons

### DISCUSSION THREAD

**Discussion by cos-p** [MiczFlor/RPi-Jukebox-RFID#2189] [Q&A] [answered]
Hi, 

I'm pretty new to this. I've got future3 main running on my pi zero 2. 

I can't really figure out how to get the USB buttons to work. I know i'm not giving a lot of info, but is there something really obvious i'm missing? 

 if i read the logs i can see that my usb device is being recognised, but no presses are registered.  in v2 i can see there's some set up scripts for them. I also tried following https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/1495 and copied over the files from v2; that didnt really work. i'm getting evdev errors. 
 
 Thank you and happy new year! 

Update: nevermind, found the features list and saw that it's not supposed to work yet.

**Comment by s-martin** [comment, marked as answer]
Hi,

#1495 works not for future3.

for future3 there's a pull request, which provides the functionality you need:

https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/1943

There's some polishing planned, that's why it's not merged yet.

But you could try to manually adapt your installation.

If you have any direct questions you can also use our chat https://matrix.to/#/#phoniebox_community:matrix.org
